platforms/android/src/org/crosswalk/engine/XWalkCordova6Interface.java

```diff
@@ -0,0 +1,34 @@
+/*
+       Licensed to the Apache Software Foundation (ASF) under one
+       or more contributor license agreements.  See the NOTICE file
+       distributed with this work for additional information
+       regarding copyright ownership.  The ASF licenses this file
+       to you under the Apache License, Version 2.0 (the
+       "License"); you may not use this file except in compliance
+       with the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+       Unless required by applicable law or agreed to in writing,
+       software distributed under the License is distributed on an
+       "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+       KIND, either express or implied.  See the License for the
+       specific language governing permissions and limitations
+       under the License.
+*/
+
+package org.crosswalk.engine;
+
+import android.webkit.ValueCallback;
+
+/**
+ * Interface for implementing methods in Cordova 6 without breaking
+ * compatibility with older versions of Cordova.
+ *
+ * Not exposed outside of the package.
+ */
+interface XWalkCordova6Interface {
```

Comment:
It's too complex to keep compatibility  with adding new interface. It's better removing the annotation "Override" and adding annotation manually like below

> /**
     * This API is used in Cordova-Android 6.0.0  override from      
     * CordovaWebViewEngine.java
     * @since Cordova 6.0
     */
public void evaluateJavascript(String js, ValueCallback<String> callback) {
         webView.evaluateJavascript(js, callback);
 }